docs/reference/client.md

```diff
@@ -136,7 +136,7 @@ Examples are a simple `name to input` dictionary.
 
 #### `kill_container(self)`
 
-#### `server_predict(self, request: dict)`
+#### `predict(self, request: dict)`
```

Comment:
I don't think this is the right API documentation. Your new API now takes `use_docker` as a keyword. Does this need to also get updated to match the correct signature?

Also, can we autogenerate this file from docstrings by any chance?